internal/configs/sharding/localnet.go

```diff
@@ -19,11 +19,11 @@ type localnetSchedule struct{}
 const (
 	localnetV1Epoch = 1
 
-	localnetEpochBlock1      = 10
-	localnetBlocksPerEpoch   = 5
-	localnetBlocksPerEpochV2 = 10
+	localnetEpochBlock1      = 5
```

Comment:
why change the localnet settings?

Reply:
commit by mistake. this is my local test configuration, to speed up the test.